### PR TITLE
Add tkn clustertask describe Command

### PR DESCRIPTION
**Pull request by danielhelfand**
Part of #594 

Adding a clustertask describe command. Follows a similar pattern to `tkn task desc`. May need to rebase this based on #611 and add test for unnamed steps. 

Can also switch to golden file method for testing.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [x] Run the code checkers with `make check`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

# Release Notes

```
Add tkn ct describe command to describe a clustertask
```

### CODE REVIEW

File: pkg/cmd/clustertask/describe_test.go [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+`,
+		},
+		{
+			name:        "Describe clustertask missing param default one of everything",
+			command:     []string{"describe", "clustertask-one-everything"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   false,
+			want: `Name:   clustertask-one-everything
```
Comment (vdemeester) [inline pkg/cmd/clustertask/describe_test.go]: Those are not needed anymore :angel:

File: pkg/cmd/clustertask/describe_test.go [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+ No taskruns
+`,
+		},
+		{
+			name:        "Describe full clustertask multiple taskruns",
+			command:     []string{"describe", "clustertask-full"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   false,
+			want: `Name:   clustertask-full
```
Comment (vdemeester) [inline pkg/cmd/clustertask/describe_test.go]: Those are not needed anymore :angel:

File: pkg/cmd/clustertask/describe_test.go [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+			wantError:   true,
+			want:        "clustertasks.tekton.dev \"notexist\" not found",
+		},
+		{
+			name:        "Describe clustertask with name only",
+			command:     []string{"describe", "clustertask-justname"},
+			input:       seeds[0],
+			inputStream: nil,
+			wantError:   false,
+			want: `Name:   clustertask-justname
```
Comment (vdemeester) [inline pkg/cmd/clustertask/describe_test.go]: Those are not needed anymore :angel:

Comment (danielhelfand) [reply pkg/cmd/clustertask/describe_test.go]: Yeah, just realized! I'll take them out.

**Pull request merged**